### ISSUE TITLE
Dynamically add asterisks to required field when `required` prop is true

### ISSUE THREAD

**Issue by inveterateliterate** [LaunchPadLab/lp-components#284]
related: https://github.com/LaunchPadLab/lp-form/issues/31

**Comment by inveterateliterate** [comment]
can also have a prop that can take in a component for the "required indicator" - reach goal; defaults to asterisk

**Comment by chawes13** [comment]
The asterisk or character that is added should be included in a span that has aria-hidden=true.

```javascript
<label htmlFor="personalDetails.email">Email<span aria-hidden="true">*</span></label>
```

**Comment by chawes13** [comment]
@inveterateliterate Got a target date for this? Would be useful for AZ

**Comment by inveterateliterate** [comment]
is end of year ok @chawes13 ?

**Comment by chawes13** [comment]
Perfect!

**Comment by inveterateliterate** [comment]
@chawes13 do screen readers read that a field is required already  if `required` is true on the input element?

**Comment by chawes13** [comment]
@inveterateliterate yep!

**Comment by dpikt** [comment]
Resolved in #310
